### PR TITLE
Add PHPUnit service container.

### PR DESCRIPTION
**Pull request by bkildow**

### CODE REVIEW

File: generators/environment/templates/docker/build.devcloud.yml [hunk 1]
```diff
@@ -145,3 +154,7 @@ services:
       NPM_CONFIG_CACHE: /root/.cache/npm
       COMPOSER_CACHE_DIR: /root/.cache/composer
       <% if (proxy.exists) { %>PROXY_VIRTUAL_HOST: ${DOCKER_ENV:-local}-<%= host.devcloud %><% } %>
+      # PHPUnit variables.
+      SIMPLETEST_DB: mysql://admin:admin@db/<%= machineName %>_drupal
+      SIMPLETEST_URL: <%= host.local %>
```
Comment (grayside) [inline generators/environment/templates/docker/build.devcloud.yml]: host.local is not correct here. See the other references to the URL in this file.

Comment (bkildow) [inline generators/environment/templates/docker/build.devcloud.yml]: Fixed.

**Comment by mike-potter** [comment]
This looks good to me and very useful.

**Comment by jhedstrom** [comment]
This looks great! Thanks!

**Pull request merged**